content/en/api/users/users_get_code.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Get user
 type: apicode
-order: 26.2
+order: 27.2
```

Comment:
Don't know about these order changes, but on https://docs-staging.datadoghq.com/kaylyn/synthetics-api/api/, Synthetics doesn't appear at the right place in the menu (to follow the alphabetical order, it should be before tags, not before tracing.

Reply:
🤦‍♀️this is such a good catch